adala/skills/collection/classification.py

```diff
@@ -29,15 +29,7 @@ def validate_labels(self):
                 )
 
             self.field_schema[labels_field] = {
-                "type": "array",
-                "items": {"type": "string", "enum": labels},
+                "type": "string",
+                "description": "The classification label.",
+                "enum": labels
```

Comment:
have you tried running the server tests? wondering if the server payload from LSE needs to be updated to account for this

Reply:
Checked with the live server
<img width="1708" alt="image" src="https://github.com/user-attachments/assets/5436a08c-18b5-4173-8286-7c7834fdda10">

Skipping one test as it fails in the master branch